docs/en/end-user/components/more.md

```diff
@@ -2,8 +2,29 @@
 title:  Want More?
 ---
 
-Components in KubeVela are designed to be brought by users.
+Components in KubeVela are designed to be brought by users. You could design one by yourself, or try some maintained by KubeVela developers.
 
+## 1. Get from capability center
+
+KubeVela allows you to explore capabilities maintained by other developer.
+There are two command in kubectl vela plugin: `comp` and `trait`.
+
+In case you haven't install kubectl vela plugin: see [this](../../kubectl-plugin).
+
+For example, let's try the list all availible components in default cap center:
+
+```shell
+$ kubectl vela comp
+Showing components from default cap center:https://github.com/oam-dev/catalog/tree/master/registry
+NAME              	CENTER	TYPE               	DEFINITION                 	STATUS	APPLIES-TO
+cloneset          	      	componentDefinition	clonesets.apps.kruise.io   	      	[]        
+kruise-statefulset	      	componentDefinition	statefulsets.apps.kruise.io	      	[]        
+openfaas          	      	componentDefinition	functions.openfaas.com     	      	[]
+```
```

Comment:
how to install the caps in remote registry?

Reply:
Haven't implement that feature in plugin, maybe before I implement that, the doc should lead people to vela CLI?

Reply:
you should implement in this PR , this is the essential part.

Reply:
I have implement that.